client/public/loginPage/loginPage.css

```diff
@@ -0,0 +1,94 @@
+@import "https://use.fontawesome.com/releases/v5.5.0/css/all.css";
+
+html {
+    height: 100%;
+}
+
+body {
+    background: linear-gradient(to right, #11998e 0%, #38ef7d 100%);
+    font-family: sans-serif;
+    height: inherit;
+}
+
+.container {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    height: inherit;
+}
+
+.login-container {
+    padding: 50px;
+    border: 2px solid;
+    border-image-slice: 2;
+    border-image-source: linear-gradient(to right, #11998e 0%, #38ef7d 100%);
+}
+
+.login-container h1 {
+    border-bottom: 4px solid #007250;
+    margin-bottom: 20px;
+    text-align: center;
+}
+
+.login-container .logo {
+    animation: wiggle 2.5s infinite;
+    height: 10em;
+    margin-left: 30%;
+}
+
+.text-box {
+    border-bottom: 1px solid #007250;
+    font-size: 20px;
+    margin: 8px 0;
+    overflow: auto;
+    padding: 8px 0;
+}
+
+.text-box i {
+    float: left;
+    text-align: center;
+    width: 26px;
+}
+
+.text-box input {
+    background: none;
```

Comment:
what do you need the background none for? none is default

Reply:
I think it inherited some other property because I checked and its not he's default